client/my-sites/plugins/single-plugin.jsx

```diff
@@ -0,0 +1,438 @@
+	);
+	const sitePlugin = useSelector(
+		( state ) => selectedSiteId && getPluginOnSite( state, selectedSiteId, props.pluginSlug )
+	);
+	const userCanManagePlugins = useSelector( ( state ) =>
+		selectedSiteId
+			? canCurrentUser( state, selectedSiteId, 'manage_options' )
+			: canCurrentUserManagePlugins( state )
+	);
+
+	const isWpcom = selectedSite && ! isJetpackSite;
+	const analyticsPath = selectedSite ? '/plugins/:plugin/:site' : '/plugins/:plugin';
+
+	const isPluginInstalledOnsite = ! requestingPluginsForSites ? !! sitePlugin : null;
+
+	const fullPlugin = {
+		...plugin,
+		...wporgPlugin,
+	};
+
+	useEffect( () => {
+		if ( ! isFetched ) {
+			dispatch( wporgFetchPluginData( props.pluginSlug ) );
+		}
+	}, [ isFetched ] );
+
+	const existingPlugin = useMemo( () => {
+		if ( isFetching || ! isFetched ) {
+			return 'unknown';
+		}
+		if ( fullPlugin && fullPlugin.fetched ) {
+			return true;
+		}
+
+		// If the plugin has at least one site then we know it exists
+		const pluginSites = Object.values( fullPlugin.sites );
+		if ( pluginSites && pluginSites[ 0 ] ) {
+			return true;
+		}
+
+		if ( requestingPluginsForSites ) {
+			return 'unknown';
+		}
+
+		return false;
+	}, [ isFetching, isFetched, fullPlugin, requestingPluginsForSites ] );
+
+	const getNavigationItems = () => {
+		// ToDo:
+		// - add "Search Results" breadcrumb if prev page was search results
+		// - change the first breadcrumb if prev page wasn't plugins page (eg activity log)
+		const navigationItems = [
+			{ label: translate( 'Plugins' ), href: `/plugins/${ selectedSite?.slug || '' }` },
+			{ label: fullPlugin.name },
+		];
+
+		return navigationItems;
+	};
+
+	const getPageTitle = () => {
+		return translate( '%(pluginName)s Plugin', {
+			args: { pluginName: fullPlugin.name },
+			textOnly: true,
+			context: 'Page title: Plugin detail',
+		} );
+	};
+
+	if ( ! isRequestingSites && ! userCanManagePlugins ) {
+		return <NoPermissionsError title={ getPageTitle() } />;
+	}
+
+	if ( existingPlugin === 'unknown' ) {
+		return <PluginPlaceholder />;
+	}
+
+	if ( existingPlugin === false ) {
+		return <PluginDoesNotExistView />;
+	}
+
+	return (
+		<MainComponent wideLayout>
+			<DocumentHead title={ getPageTitle() } />
+			<PageViewTracker path={ analyticsPath } title="Plugins > Plugin Details" />
+			<QueryJetpackPlugins siteIds={ siteIds } />
+			<SidebarNavigation />
+			<FixedNavigationHeader navigationItems={ getNavigationItems() } />
+			<PluginNotices pluginId={ fullPlugin.id } sites={ sites } plugins={ [ fullPlugin ] } />
+
+			<div className="single-plugin__page">
+				<div className="single-plugin__layout single-plugin__top-section">
+					<div
+						className={ classNames( 'single-plugin__layout-col', 'single-plugin__layout-col-left', {
+							'no-cta ': ! shouldDisplayCTA(
+								selectedSite,
+								props.pluginSlug,
+								isPluginInstalledOnsite
+							),
+						} ) }
+					>
+						<div className="single-plugin__header">
```

Comment:
Can we add the short description and the tags in the header too?
![SS 2021-11-09 at 18 18 56](https://user-images.githubusercontent.com/12430020/140963121-93622065-ffca-4960-83fd-c2235f347546.png)

Reply:
> Can we add the short description and the tags in the header too?

Checking the disponibility of those informations

> padding

I will work on it now

> stats( developer, ratings) should be 2 column. I am not sure why you used table there to be honest. Is there something that can't be achieved with flexbox?

No strong reason to be using table, I just tough it would make more sense semantically, but considering the mobile version I will use flexbox.